packages/@react-aria/tabs/docs/useTab.mdx

```diff
@@ -0,0 +1,195 @@
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-aria/tabs';
+import {HeaderInfo, FunctionAPI, TypeContext, InterfaceType} from '@react-spectrum/docs';
+import packageData from '@react-aria/tabs/package.json';
+import Anatomy from './anatomy.svg';
+
+```jsx import
+import {useTab, useTabList, useTabPanel} from '@react-aria/tabs';
+import {useTabListState} from '@react-stately/tabs';
+```
+
+---
+category: Navigation
+keywords: [tabs, aria]
+---
+
+# useTabList
+
+<p>{docs.exports.useTab.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['useTabList', 'useTab', 'useTabPanel']}
+  sourceData={[
+    {type: 'W3C', url: 'https://www.w3.org/TR/wai-aria-practices/#tabpanel'}
+  ]} />
+
+## API
+
+<FunctionAPI function={docs.exports.useTabList} links={docs.links} />
+<FunctionAPI function={docs.exports.useTab} links={docs.links} />
+<FunctionAPI function={docs.exports.useTabPanel} links={docs.links} />
+
+## Features
+
+Tabs provide a list of tabs that a user can select from to change the panels contents. `useTab`, `useTabList`, and `useTabPanel` help implement these in an accessible way.
+
+* Support for mouse, touch, and keyboard interactions on tabs
+* Support for LTR and RTL keyboard navigation
+* Support for disabled tabs
+* Support for accessibly linking Tabs and the associated Tab Panel
+
+
+## Anatomy
+
+<Anatomy />
+
+Tabs consist of a tablist with one or more visually separated tabs. Each tab has associated content and only the selected tab's content is shown.
+Each Tab can be clicked, tapped, or navigated to via arrow keys. Depending on the mode, the tab can be selected by receiving keyboard focus or it can be selected with the <kbd>Enter</kbd> key.
+
+`useTabList` returns props to spread onto any element containing the tabs:
+
+<TypeContext.Provider value={docs.links}>
+  <InterfaceType properties={docs.links[docs.exports.useTabList.return.id].properties} />
+</TypeContext.Provider>
+
+`useTab` returns props to be spread onto each individual tab:
+
+<TypeContext.Provider value={docs.links}>
+  <InterfaceType properties={docs.links[docs.exports.useTab.return.id].properties} />
+</TypeContext.Provider>
+
+`useTabPanel` returns props to spread onto the container for the tab content:
+
+<TypeContext.Provider value={docs.links}>
+  <InterfaceType properties={docs.links[docs.exports.useTabPanel.return.id].properties} />
+</TypeContext.Provider>
+
+## Example
+
+This example displays a basic list of tabs. The currently selected Tab get a tabindex of 0 while the rest are set to -1 ensuring that the Tab list is only a single tab stop. The selected tab gets a different style so it's obvious which one is currently selected. Our hooks handle the aria properties that associate the tabs with their tab panel for screen readers. In the tab panel itself, we only render the currently selected item.
+
+```tsx example
```

Comment:
If the tabs component in the last example is the same, you can change this to:
```suggestion
```tsx example export=true
```
and get rid of the collapsed dupe code

see useColorField.mdx for an example (wish I could link to the proper place but you have to look at the raw file)
https://github.com/adobe/react-spectrum/blob/1e2b7f280a04ead83e21b27fa580eb189e5f4186/packages/%40react-aria/color/docs/useColorField.mdx

Reply:
I'd forgotten about this one, thank you!